debian/control

```diff
@@ -2,11 +2,60 @@ Source: ros-noetic-robot-localization
 Section: misc
 Priority: optional
 Maintainer: Tom Moore <ayrton04@gmail.com>
```

Comment:
Hmm, at this point, should we update the maintainer to be `developers@greenzie.com`?

Reply:
we should. good catch